app/services/utils/pdf_generator.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require_relative '../../views/helpers/line_break_helper.rb'
```

Comment:
Style/RedundantFileExtensionInRequire: Redundant `.rb` file extension detected. (https://rubystyle.guide#no-explicit-rb-to-require)

Reply:
I suggest to add the `helper_folder` in `config.eager_load_paths` (in `/config/application.rb`)